src/lib/x509/x509_ext.cpp

```diff
@@ -764,6 +768,51 @@ void CRL_Issuing_Distribution_Point::decode_inner(const std::vector<uint8_t>& bu
    BER_Decoder(buf).decode(m_distribution_point).verify_end();
 }
 
+void TNAuthList::Entry::encode_into(DER_Encoder&) const {
+   throw Botan::Not_Implemented("TNAuthList extension serialization is not supported");
```

Comment:
```suggestion
   throw Botan::Not_Implemented("TNAuthList extension entry serialization is not supported");
```

... unlikely that we'll ever see this error message. Just making it unique for good measure. `TNAuthList::encode_inner()` used the exact same error string, making it harder to interpret error logs.